library/src/main/java/bio/terra/landingzone/common/utils/LandingZoneFlightBeanBag.java

```diff
@@ -103,4 +105,12 @@ public AzureCustomerUsageConfiguration getAzureCustomerUsageConfiguration() {
   public AzureCredentialsProvider getAzureCredentialsProvider() {
     return azureCredentialsProvider;
   }
+
+  public void setArmManagers(ArmManagers armManagers) {
```

Comment:
This sticks out a bit from the rest of the values here, because it's not autowired in.
But this is where resources live that can't easily be serialized into the working map (and the ArmManagers cannot).

We can't initialize ArmManagers automatically, because it requires the output of earlier steps.